*** /home/fab/Desktop/Universibo/forum/forum/page_header.php	2005-06-26 13:47:10.000000000 +0200
--- ./page_header.php	2005-06-26 12:07:38.000000000 +0200
***************
*** 73,76 ****
--- 73,78 ----
  if ( $userdata['session_logged_in'] )
  {
+ 	$method		= 'post';
+ 	$u_action	= '../v2.php?do=Logout'; 	
  	$u_login_logout = 'login.'.$phpEx.'?logout=true&amp;sid=' . $userdata['session_id'];
  	$l_login_logout = $lang['Logout'] . ' [ ' . $userdata['username'] . ' ]';
***************
*** 78,83 ****
--- 80,88 ----
  else
  {
+ 	$method		= 'get';
  	$u_login_logout = 'login.'.$phpEx;
+ 	$u_action	= $u_login_logout;
  	$l_login_logout = $lang['Login'];
+ 	
  }
  
***************
*** 396,399 ****
--- 402,410 ----
  	'U_GROUP_CP' => append_sid('groupcp.'.$phpEx),
  
+ //	begin edit by evaimitico
+ 	'U_ACTION' => append_sid($u_action),
+ 	'METHOD'   => $method,
+ //	end edit by evaimitico
+ 	
  	'S_CONTENT_DIRECTION' => $lang['DIRECTION'],
  	'S_CONTENT_ENCODING' => $lang['ENCODING'],
***************
*** 401,405 ****
  	'S_CONTENT_DIR_RIGHT' => $lang['RIGHT'],
  	'S_TIMEZONE' => sprintf($lang['All_times'], $l_timezone),
! 	'S_LOGIN_ACTION' => append_sid('login.'.$phpEx),
  
  	'T_HEAD_STYLESHEET' => $theme['head_stylesheet'],
--- 412,417 ----
  	'S_CONTENT_DIR_RIGHT' => $lang['RIGHT'],
  	'S_TIMEZONE' => sprintf($lang['All_times'], $l_timezone),
! //	'S_LOGIN_ACTION' => append_sid('login.'.$phpEx),
! 	'S_LOGIN_ACTION' => '../v2.php?do=Login',
  
  	'T_HEAD_STYLESHEET' => $theme['head_stylesheet'],
