*** /home/fab/Desktop/Universibo/forum/forum/index_body.tpl	2005-06-26 13:47:15.000000000 +0200
--- ./index_body.tpl	2005-06-26 12:37:43.000000000 +0200
***************
*** 74,84 ****
  	<tr> 
  	  <td class="row1" align="center" valign="middle" height="28"><span class="gensmall">{L_USERNAME}: 
! 		<input class="post" type="text" name="username" size="10" tabindex="1"/>
! 		&nbsp;&nbsp;&nbsp;{L_PASSWORD}: 
! 		<input class="post" type="password" name="password" size="10" maxlength="32" tabindex="2"/>
! <!--		&nbsp;&nbsp; &nbsp;&nbsp;{L_AUTO_LOGIN} 
  		<input class="text" type="checkbox" name="autologin" />
  -->		&nbsp;&nbsp;&nbsp; 
! 		<input type="submit" class="mainoption" name="login" value="{L_LOGIN}" tabindex="3" />
  		</span> </td>
  	</tr>
--- 74,86 ----
  	<tr> 
  	  <td class="row1" align="center" valign="middle" height="28"><span class="gensmall">{L_USERNAME}: 
! <!--		<input class="post" type="text" name="username" size="10" />
! -->		<input class="post" type="text" name="f1_username" size="10" maxlength="25" tabindex="1" />
! 		&nbsp;&nbsp;&nbsp;{L_PASSWORD}:  
! 		<input class="post" type="password" name="f1_password" size="10" maxlength="25" tabindex="1" />
! <!--		<input class="post" type="password" name="password" size="10" maxlength="32" />
! 		&nbsp;&nbsp; &nbsp;&nbsp;{L_AUTO_LOGIN} 
  		<input class="text" type="checkbox" name="autologin" />
  -->		&nbsp;&nbsp;&nbsp; 
! 		<input type="submit" class="mainoption" name="f1_submit" value="{L_LOGIN}" tabindex="1" />
  		</span> </td>
  	</tr>
