*** /home/fab/Desktop/Universibo/forum/forum/login_body.tpl	2005-06-26 13:47:16.000000000 +0200
--- ./login_body.tpl	2005-06-26 12:11:48.000000000 +0200
***************
*** 20,24 ****
  			<td width="45%" align="right"><span class="gen">{L_USERNAME}:</span></td>
  			<td> 
! 			  <input type="text" name="username" size="25" maxlength="40" value="{USERNAME}" tabindex="1" />
  			</td>
  		  </tr>
--- 20,25 ----
  			<td width="45%" align="right"><span class="gen">{L_USERNAME}:</span></td>
  			<td> 
! 			 <!-- <input type="text" name="username" size="25" maxlength="40" value="{USERNAME}" />-->
! 			 <input type="text" name="f1_username" size="25" maxlength="40" value="{USERNAME}" />
  			</td>
  		  </tr>
***************
*** 26,34 ****
  			<td align="right"><span class="gen">{L_PASSWORD}:</span></td>
  			<td> 
! 			  <input type="password" name="password" size="25" maxlength="32" tabindex="2"  />
  			</td>
  		  </tr>
  		  <tr align="center"> 
! 			<td colspan="2">{S_HIDDEN_FIELDS}<input type="submit" name="login" class="mainoption" value="{L_LOGIN}" tabindex="3" /></td>
  		  </tr>
  		  <tr align="center"> 
--- 27,37 ----
  			<td align="right"><span class="gen">{L_PASSWORD}:</span></td>
  			<td> 
! <!--			  <input type="password" name="password" size="25" maxlength="32" />-->
! 			  <input type="password" name="f1_password" size="25" maxlength="32" />
  			</td>
  		  </tr>
  		  <tr align="center"> 
! 			<!--<td colspan="2">{S_HIDDEN_FIELDS}<input type="submit" name="login" class="mainoption" value="{L_LOGIN}" /></td>-->
! 			<td colspan="2">{S_HIDDEN_FIELDS}<input type="submit" name="f1_submit" class="mainoption" value="{L_LOGIN}" /></td>
  		  </tr>
  		  <tr align="center"> 
