*** /home/fab/Desktop/Universibo/forum/forum/overall_header.tpl	2005-06-26 13:47:13.000000000 +0200
--- ./overall_header.tpl	2005-06-26 11:48:53.000000000 +0200
***************
*** 32,35 ****
--- 32,54 ----
  }
  
+ /* add by evaimitico */
+  
+  #f2{
+  display: inline;
+  }
+  
+  #f2 input{
+  background-color: white;
+  color: rgb(56, 109, 206);
+  padding: 0;
+  margin: 0;
+  border: 0 none;
+  }
+  
+  #f2 input:hover{ text-decoration: underline; color: rgb(56, 109, 206); 
+  cursor: pointer;}
+  
+  /* end add*/
+ 
  /* General font families for common tags */
  font,th,td,p { font-family: {T_FONTFACE1} }
***************
*** 56,60 ****
  td.rowpic {
  		background-color: {T_TD_COLOR2};
! /*<!-- BRAIN		background-image: url(templates/subSilver/images/{T_TH_CLASS3}); -->*/
  		background-repeat: repeat-y;
  }
--- 75,79 ----
  td.rowpic {
  		background-color: {T_TD_COLOR2};
! <!-- BRAIN		background-image: url(templates/subSilver/images/{T_TH_CLASS3}); -->
  		background-repeat: repeat-y;
  }
***************
*** 64,72 ****
  	color: {T_FONTCOLOR3}; font-size: {T_FONTSIZE2}px; font-weight : bold; 
  	background-color: {T_BODY_LINK}; height: 25px;
! /*<!-- BRAIN	background-image: url(templates/subSilver/images/{T_TH_CLASS2}); -->*/
  }
  
  td.cat,td.catHead,td.catSides,td.catLeft,td.catRight,td.catBottom {
! /*<!-- BRAIN			background-image: url(templates/subSilver/images/{T_TH_CLASS1}); -->*/
  			background-color:{T_TR_COLOR3}; border: {T_TH_COLOR3}; border-style: solid; height: 28px;
  }
--- 83,91 ----
  	color: {T_FONTCOLOR3}; font-size: {T_FONTSIZE2}px; font-weight : bold; 
  	background-color: {T_BODY_LINK}; height: 25px;
! <!-- BRAIN	background-image: url(templates/subSilver/images/{T_TH_CLASS2}); -->
  }
  
  td.cat,td.catHead,td.catSides,td.catLeft,td.catRight,td.catBottom {
! 			background-image: url(templates/subSilver/images/{T_TH_CLASS1});
  			background-color:{T_TR_COLOR3}; border: {T_TH_COLOR3}; border-style: solid; height: 28px;
  }
***************
*** 240,244 ****
  					</tr>
  					<tr>
! 						<td height="25" align="center" valign="top" nowrap="nowrap"><span class="mainmenu">&nbsp;<a href="{U_PROFILE}" class="mainmenu"><img src="templates/subSilver/images/icon_mini_profile.gif" width="12" height="13" border="0" alt="{L_PROFILE}" hspace="3" />{L_PROFILE}</a>&nbsp; &nbsp;<a href="{U_PRIVATEMSGS}" class="mainmenu"><img src="templates/subSilver/images/icon_mini_message.gif" width="12" height="13" border="0" alt="{PRIVATE_MESSAGE_INFO}" hspace="3" />{PRIVATE_MESSAGE_INFO}</a>&nbsp; &nbsp;<a href="{U_LOGIN_LOGOUT}" class="mainmenu"><img src="templates/subSilver/images/icon_mini_login.gif" width="12" height="13" border="0" alt="{L_LOGIN_LOGOUT}" hspace="3" />{L_LOGIN_LOGOUT}</a>&nbsp;</span></td>
  					</tr>
  				</table></td>
--- 259,263 ----
  					</tr>
  					<tr>
! 						<td height="25" align="center" valign="top" nowrap="nowrap"><span class="mainmenu">&nbsp;<a href="{U_PROFILE}" class="mainmenu"><img src="templates/subSilver/images/icon_mini_profile.gif" width="12" height="13" border="0" alt="{L_PROFILE}" hspace="3" />{L_PROFILE}</a>&nbsp; &nbsp;<a href="{U_PRIVATEMSGS}" class="mainmenu"><img src="templates/subSilver/images/icon_mini_message.gif" width="12" height="13" border="0" alt="{PRIVATE_MESSAGE_INFO}" hspace="3" />{PRIVATE_MESSAGE_INFO}</a>&nbsp; &nbsp;<form action="{U_ACTION}" name="f2" id="f2" method="{METHOD}" class="mainmenu"><img src="templates/subSilver/images/icon_mini_login.gif" width="12" height="13" border="0" alt="{L_LOGIN_LOGOUT}" hspace="3" /><input class="submit" name="f2_submit" value="{L_LOGIN_LOGOUT}" type="submit" /></form>&nbsp;</span></td>
  					</tr>
  				</table></td>
